app/src/main/java/co/runloop/influencer/model/Resource.java

```diff
@@ -0,0 +1,54 @@
+package co.runloop.influencer.model;
+
+import io.reactivex.annotations.NonNull;
+
+public class Resource<T> {
+
+    public enum Status {
+        Completed,
+        Progress,
+        Error
+    }
+
+    private T data;
+    private Status status;
+    private Throwable error;
```

Comment:
Maybe annotate the members as well? And/or the getters?

Reply:
Do you mean to add some documentation/explanation ?

Reply:
No, I meant with `@NonNull` and `@Nullable`